docker/install_deps.sh

```diff
@@ -47,13 +47,22 @@ if [[ "${TARGETARCH}" == "arm" ]]; then
 fi
 
 # ffmpeg -> arm64
+if command -v ffmpeg >/dev/null 2>&1; then
```

Comment:
in which cases would HAS_FFMPEG be 0 for this?

Reply:
I'm not familiar with all if\else logic for all platforms, so ensuring here my part will work as it should be without breaking other's people work